components/constellation/constellation.rs

```diff
@@ -1929,15 +1920,15 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
     }
 
     fn handle_get_pipeline(&mut self, frame_id: Option<FrameId>,
-                           resp_chan: IpcSender<Option<(PipelineId, bool)>>) {
+                           resp_chan: IpcSender<Option<PipelineId>>) {
         let frame_id = frame_id.unwrap_or(self.root_frame_id);
         let current_pipeline_id = self.frames.get(&frame_id)
             .map(|frame| frame.current.pipeline_id);
         let current_pipeline_id_loaded = current_pipeline_id
-            .map(|id| (id, true));
+            .map(|id| id);
```

Comment:
Sorry for the late question: what this `map` call does? As I read it, the callback is an identity function, so it just copies the array.

NB: I am a beginner in Rust, so I may be wrong, don't hesitate to correct me :).